src/pyOpenMS/tests/unittests/test_Chemistry.py

```
@@ -0,0 +1,570 @@
+import pyopenms
+import copy
+import os
+
+from pyopenms import String as s
+import numpy as np
+import pandas as pd
+
+print("IMPORTED ", pyopenms.__file__)
+
+try:
+    long
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)